providers/common/ciphers/aes.c

```diff
@@ -54,12 +54,12 @@ static int aes_einit(void *vctx, const unsigned char *key, size_t keylen,
     PROV_AES_KEY *ctx = (PROV_AES_KEY *)vctx;
 
     if (!PROV_AES_KEY_generic_init(ctx, iv, ivlen, 1)) {
-        /* PROVerr already called */
+        /* ERR_raise( already called */
```

Comment:
Your substitution regexp left an artifact 😉

Reply:
Yes it was a very primitive version of regex :)
Fixed.